### PR TITLE
Add option to inject community or custom plugins

### PR DESCRIPTION
**Pull request by pipo02mix**
Extend the deployment and values to be able to define new plugins.

### CODE REVIEW

File: helm/kong-app/templates/plugins-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  name: kong-plugins-{{ .name }}
+  labels:
+    component: app
+data:
+{{ range (.files) }}
+  {{ .filename }}: |
+{{ .filecontent | indent 4 }}
+{{ end }}
+---
+{{ end }}
```
Comment (rossf7) [inline helm/kong-app/templates/plugins-configmap.yaml]: Misses the newline.

File: helm/kong-app/values.yaml [hunk 2]
```diff
@@ -397,3 +397,27 @@ ingressController:
...
+  #         end
+
+  #         return MyPluginHandler
+  #     - filename: schema.lua
+  #       filecontent: |-
+  #         return {
+  #           fields = {
+  #               message = { type = "string" }
+  #           }
+  #         }
```
Comment (rossf7) [inline helm/kong-app/values.yaml]: Same here.

File: helm/kong-app/templates/plugins-configmap.yaml (hunk 1 above)
Comment (pipo02mix) [inline helm/kong-app/templates/plugins-configmap.yaml]:
```suggestion
{{ end }}

```

File: helm/kong-app/values.yaml (hunk 2 above)
Comment (pipo02mix) [inline helm/kong-app/values.yaml]:
```suggestion
  #         }
  
```

**Pull request merged**